dist/Dockerfile.deploy/Dockerfile

```
@@ -1,3 +1,11 @@
+FROM cincinnati:build as builder
+# build
+COPY . .
+RUN cargo build --release && \
+    mkdir -p /opt/cincinnati/bin && \
+    cp -rvf target/release/graph-builder /opt/cincinnati/bin && \
+    cp -rvf target/release/policy-engine /opt/cincinnati/bin && \
+
```

Either
```suggestion
    :
```
or remove the `&& \` in the previous line

---

Yep, fixed, thanks